docs/whats_new/v0.7.0.md

```diff
@@ -0,0 +1,145 @@
+
+### Deprecations
+
+* `autogluon.vision` namespace is deprecated. @bryanyzhu (#2790, #2819, #2832)
+* `autogluon.text` namespace is deprecated. @sxjscience @Innixma (#2695, #2847)
+
+
+
+## Tabular
+
+1) TabularPredictor’s inference speed has been heavily optimized, with an average 250% speedup for real-time inference. This means that TabularPredictor can satisfy <10 ms end-to-end latency on many datasets when using `infer_limit`, and the `high_quality` preset can satisfy <100 ms end-to-end latency on many datasets by default.
+2) TabularPredictor’s `"multimodal"` hyperparameter preset now leverages the full capabilities of MultiModalPredictor, resulting in stronger performance on datasets containing a mix of tabular, image, and text features.
+
+### Performance Improvements
+
+- Upgraded versions of all dependency packages to use the latest releases. @Innixma (#2823, #2829, #2834, #2887, #2915)
+- Accelerated ensemble inference speed by 150% by removing TorchThreadManager context switching. @liangfu (#2472)
+- Accelerated FastAI neural network inference speed by 100x+ and training speed by 10x on datasets with many features. @Innixma (#2909)
+- (From 0.6.1) Avoid unnecessary DataFrame copies to accelerate feature preprocessing by 25%. @liangfu (#2532)
+- (From 0.6.1) Refactor `NN_TORCH` model to be dataset iterable, leading to a 100% inference speedup. @liangfu (#2395)
+- MultiModalPredictor is now used as a member of the ensemble when `TabularPredictor.fit` is passed `hyperparameters="multimodal"`. @Innixma (#2890)
+
+### API Enhancements
+
+- Added `predict_multi` and `predict_proba_multi` methods to `TabularPredictor` to efficiently get predictions from multiple models. @Innixma (#2727)
+- Allow label column to not be present in `leaderboard` calls when scoring is disabled. @Innixma (#2912)
+
+### Deprecations
+
+- Added a deprecation warning when calling `predict_proba` with `problem_type="regression"` @Innixma (#2684)
+
+### Bug Fixes / Doc Improvements
+
+- Fixed incorrect time_limit estimation in `NN_TORCH` model. @Innixma (#2909)
+- Fixed error when fitting with only text features. @Innixma (#2705)
+- Fixed error when `calibrate=True, use_bag_holdout=True` in `TabularPredictor.fit`. @Innixma (#2715)
+- Fixed error when tuning `n_estimators` with RandomForest / ExtraTrees models. @Innixma (#2735)
+- Fixed missing onnxruntime dependency on Linux/MacOS when installing optional dependency `skl2onnx`. @liangfu (#2923)
+- Fixed edge-case RandomForest error on Windows. @yinweisu (#2851)
+- Added improved logging for `refit_full`. @Innixma (#2913)
+- Added `compile_models` to the deployment tutorial. @liangfu (#2717)
+- Various internal code refactoring. @Innixma (#2744, #2887)
+- Various doc and logging improvements. @Innixma (#2668)
+- Added JupyterLite/Pyscript support for Tabular to run training and prediction in browser environment. @yzhliu (#2384)
```

Comment:
Is this actually released yet? If not, it would be confusing mentioning here

Reply:
Talked with @yzhliu and we decided to remove JupyterLite for now